core/src/test-core/java/com/radixdlt/modules/FunctionalRadixNodeModule.java

```diff
@@ -123,7 +120,8 @@ static NodeStorageConfig file(File folder) {
 
   public enum SafetyRecoveryConfig {
     MOCKED,
-    BERKELEY_DB,
+    REAL,
+    REAL_WITH_ADDRESS_BOOK
```

Comment:
Address book is completely unrelated to the safety store. We shouldn't add this as an option.

Reply:
Reworked.